### DISCUSSION TITLE
Alternative to removed JObject-based constructor for DependencyGraphSpec

### DISCUSSION THREAD

**Discussion by 7enderhead** [NuGet/Home#11345] [Q&A] [answered]
- Version 6 does away with the DependencyGraphSpec constructor and Load method based on the JObject Json representation
- What is the alternative recommended way to create a DependencyGraphSpec from the output generated by `dotnet msbuild /t:GenerateRestoreGraphFile /p:RestoreGraphOutputPath=...` ?

**Comment by 7enderhead** [comment]
(Now also asked [at stackoverflow](https://stackoverflow.com/q/69794233/3739381)).

**Comment by zivkan** [comment, marked as answer]
Looking through `DependencyGraphSpec.cs`, I see there's a static `Load(string path)` method: 

https://github.com/NuGet/NuGet.Client/blob/2c6141ab883f14c2dbb847b71c7d89d44a2f4fcb/src/NuGet.Core/NuGet.ProjectModel/DependencyGraphSpec.cs#L227

**Reply by 7enderhead** [reply]
As simple as that, thank you!
